try:
    x = 3
    print(x[1,2:3,4])
except:
    print('it was supposed to fail')

